text/src/autogluon/text/automm/optimization/lit_module.py

```diff
@@ -44,6 +45,7 @@ def __init__(
             custom_metric_func: Callable = None,
             test_metric: Optional[torchmetrics.Metric] = None,
             efficient_finetune: Optional[str] = None,
+            mixup_alpha: Optional[float] = 0,
```

Comment:
We need to pass a mixup_fn here, which should be initialized in `AutoMMPredictor.fit()`. Basically, `AutoMMPredictor.fit()` is where we make preparations for training. `mixup_alpha ` is a data related hyper-parameter, which is not good design to be the input of lit_module.

Reply:
Refer to https://github.com/rwightman/pytorch-image-models/blob/master/train.py#L634